versions.yaml

```diff
@@ -75,7 +75,7 @@ assets:
       url: "https://github.com/cloud-hypervisor/cloud-hypervisor"
       uscan-url: >-
         https://github.com/cloud-hypervisor/cloud-hypervisor/tags.*/v?(\d\S+)\.tar\.gz
-      version: "stable/v0.5.x"
+      version: "df794993f8abe20f829275c77fb2a52ed485f70a"
```

Comment:
Question: are we planning to bump the version of CLH when we merge this PR?

Reply:
Since this is effectively a re-vendor, please can you add details of "what changed" in CLH between these two versions in the commit that changes this file.

Reply:
Yes,  I think is fine as we are developing for master and seems to be stable, but agree that before we move current master to stable we move to a tag as previous stable releases.